reo/nested_inputs.py

```diff
@@ -994,8 +995,8 @@ def list_of_dict(input):
           "type": "float",
           "min": 0.0,
           "max": 1.0e5,
-          "default": 3013.0,  # if the default value of 3013 goes in techs.py, there is a logic to assign actual defaul cost based on 'size_class'
-          "description": "Total upfront installed costs in US dollars/kW. Determined by size_class. For the 'large' (>2MW) size_class the cost is $1,874/kW. For the 'medium commercial' size_class the cost is $4,111/kW. For the 'small commercial' size_class the cost is $4,989/kW and for the 'residential' size_class the cost is $10,792/kW "
+          "default": 3013.0,  # if the default value of 3013 goes in techs.py, there is a logic to assign actual default cost based on 'size_class'
+          "description": "Total upfront installed costs in US dollars/kW. Determined by size_class. For the 'large' (>2MW) size_class the cost is $2,239/kW. For the 'medium commercial' size_class the cost is $2,766/kW. For the 'small commercial' size_class the cost is $4,300/kW and for the 'residential' size_class the cost is $5,675/kW "
```

Comment:
@NLaws From the Pivotal story, I thought "large" class was greater than 1.5MW, as opposed to 2MW as stated here. I'm going to get clarification on the sizes/ranges from Linda today in the web tool meeting before finalizing the UI changes too.

Reply:
The size class, if not provided by the user, is determined by the average load:
https://github.com/NREL/REopt_API/blob/8a2b201479e51c09581b38f8e9db4d9632baa624/reo/validators.py#L1149
and the size class impacts the production factor based on hub height:
https://github.com/NREL/REopt_API/blob/8a2b201479e51c09581b38f8e9db4d9632baa624/reo/src/techs.py#L182
So, I think that the designation of a certain power rating based on size class has no effect in REopt. We could delete the `(>2 MW)` in the description if you think that it is misleading.